docs/cep/query-guide/functions/memcache/memcache.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Memcache
+---
+
+The `memcache` function provides a persistent cache per tenant. It is similar to [cache](../cache/), but this function has a time-to-live (ttl) parameter.
+
+## Features
+
+The following functions are allowed:
+
+- `memcache:get("key")`       - gets a value per given key
+- `memcache:put("key", "value", ttl)` - puts "key" and "value" with ttl, where ttl set by default 5000L (5 seconds)
```

Comment:
maybe better to mention that `ttl` is optional. Therefore `memcache:put` function can be as :
- `memcache:put("key", "value", ttl)`
- `memcache:put("key", "value")`

Reply:
Also, we can say that `memcache:put` function returns a "value" that saves into a cache.